composer.json

```diff
@@ -59,6 +61,11 @@
     }
   },
   "extra": {
+    "patches": {
+        "friends-of-behat/mink-extension": {
+            "setClient error": "https://github.com/FriendsOfBehat/MinkExtension/files/9480850/16.txt"
+        }
+    },
     "branch-alias": {
       "dev-master": "4.3.x-dev"
```

Comment:
you'll want to merge `master` in since this line has changed for `5.0.x` now.

Reply:
I created https://github.com/mikemadison13/drupalextension/pull/1 (against fork)